hr_timesheet_sheet/models/account_analytic_line.py

```diff
@@ -85,6 +85,10 @@ def _check_state(self):
                     _('You cannot modify an entry in a confirmed '
                       'timesheet sheet.'))
 
+    def _negative_unit_amount(self):
```

Comment:
I think it's not the best way to workaround, by default new modules should not break already-existing tests of other modules? @pedrobaeza yet this is not my call